### PR TITLE
refact(productReviews): validate average ratings/review comments data

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

- Update `productReviews` service by validate average rating / review comments data, logging an error when they are invalid;
- Makes `ProductReviewCommentsBlock` simplier by removing some safe verifications as we've increased the the trust on getting data in the right format.

## Justify why they are needed

After some discussions we thought that would make `ProductReviewCommentsBlock` simplier. Another approach would be to make sure nothing fish get's written into the database (vercel/kv), but I guess we can start with this as it's easier to implement and also easier to let us now when something is wrong with the data.

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3456** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3456" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3456?utm_source=stack-comment).

File: apps/store/src/services/productReviews/productReviews.ts [hunk 1]
```diff
@@ -14,13 +18,19 @@ export const getProductAverageRating = async (productId: string) => {
   const averageRatings = await kv.get<AverageRatingsData>(KV_AVERAGE_RATINGS_KEY)
 
   if (!averageRatings) {
-    console.warn('Could not get average ratings')
+    console.error('Could not get average ratings')
```
Comment (robinandeer) [inline apps/store/src/services/productReviews/productReviews.ts:21]: is it an error if we handle it?

Comment (guilhermespopolin) [reply apps/store/src/services/productReviews/productReviews.ts:21]: Good point 👍🏻

**Review by robinandeer — APPROVED** [review]
Looks clean -- the only concern I have is if this would add to the client bundle size... 🤔

**Comment by guilhermespopolin** [comment]
> Looks clean -- the only concern I have is if this would add to the client bundle size... 🤔

But it shouldn't right?! I mean, it's code called on the server...

**Comment by guilhermespopolin** [comment]
### Merge activity

* **Oct 19, 10:51 AM**: @guilhermespopolin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3456).

**Pull request merged**